docs/spec/index.html

```diff
@@ -1,740 +1,918 @@
+  </section>
+
+  <section class="informative">
+    <h2>Security Considerations</h2>
+
+    <p>
+      There are a number of security considerations that implementers should
+      be aware of when processing data described by this specification.
+      Ignoring or not understanding the implications of this section can
+      result in security vulnerabilities.
+    </p>
+
+    <p>
+      While this section attempts to highlight a broad set of security
+      considerations, it is not a complete list. Implementers are urged to
+      seek the advice of security and cryptography professionals when
+      implementing mission critical systems using the technology outlined in
+      this specification.
+    </p>
+
+    <section class="normative">
+      <h2>General Guidelines</h2>
       <p>
-        See terms as defined in the
-        <a href="https://w3c-ccg.github.io/traceability-vocab/#terminology">
-          Traceability Vocabulary
-        </a>
-        document.
+        As a rule, systems conforming with this specification SHOULD leverage
+        and comply with encryption and security guidelines as listed in:
+        <a href="https://csrc.nist.gov/publications/detail/fips/186/5/draft"
+          rel="nofollow">FIPS 186-5 (DRAFT)</a>,
+        <a href="https://csrc.nist.gov/publications/detail/fips/180/4/final"
+          rel="nofollow">FIPS 180-4</a>, and
+        <a href="https://csrc.nist.gov/publications/detail/fips/197/final"
+          rel="nofollow">FIPS 197</a>. This effectively means that committers
+        should be thinking along the
+        lines of SHA-256, Ed25519, and Rijndael versus other competing
+        algorithms.
       </p>
     </section>
 
-    <section
-      data-include="sections/use-case-requirements.html"
-      data-include-replace="true"
-    ></section>
-
-    <section
-      data-include="sections/authorization.html"
-      data-include-replace="true"
-    ></section>
-
-    <section
-      data-include="sections/api-spec.html"
-      data-include-replace="true"
-    ></section>
-
-    <section id="conformance"></section>
-
-    <section class="informative">
-      <h2>Interoperability Testing</h2>
+    <section class="normative">
+      <h2>Linked Data Signature Suites</h2>
       <p>
-        Participating platforms are proving actual interoperability by enrolling in continuous integration tests. These tests are based on Postman
-        collections which orchestrate the interaction of participating parties; for example, `Credential Issuance`, `Credential Verification`,
-        and `Presentation Exchange`.
-        These Postman collections are described with a set of <a href="https://github.com/w3c-ccg/traceability-interop/tree/main/docs/tutorials">tutorials</a>
-        that show how to run the tests against a particular implementation.
+        Any system conforming with this specification for interoperability
+        MUST utilize
+        <a href="https://github.com/w3c-ccg/lds-jws2020/">
```

Comment:
```suggestion
        <a href="https://github.com/w3c/vc-jws-2020">
```
The other link is deprecated